doc-content/drools-docs/src/main/asciidoc/DMN/dmn-gs-creating-drd-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='dmn-gs-creating-drd-proc']
+= Creating the traffic violations DMN decision requirements diagram (DRD)
+
+Use the DMN designer in {CENTRAL} to design the DMN decision requirements diagrams (DRDs) and define decision logic for the traffic violation project.
+
+.DRD for the Traffic Violation example
+image::dmn/dmn-traffic-violations-drd.png[]
+
+.Prerequisite
+* You have created the traffic-violations project in {CENTRAL}.
```

Comment:
Changed as suggested.

Reply:
So, naming is still kind of inconsistent throughout. Sometimes you use a dash, sometimes a space, sometimes underscore, sometimes lower case, sometimes upper case.

Search all instances of traffic violation(s), Traffic Violation(s), traffic-violation(s), and traffic_violation(s) and pick one. Here is what you have in your screenshots, so I suggest applying these:

* Project name: `traffic_violations`
* DMN file name: `Traffic Violation`
* Scenario file name: `Violation Scenarios`
* Suggested generic reference: traffic violations (lower case, plural, no formatting, otherwise, if referring specifically to either the project or DMN or scenario name, use the exact name as shown in the UI, and bold. I recommend using the generic, unbolded, lower case form as much as possible.)